src/protocol_key.c

```diff
@@ -32,7 +32,7 @@
 #include "xalloc.h"
 
 void send_key_changed(meshlink_handle_t *mesh) {
-	send_request(mesh, mesh->everyone, "%d %x %s", KEY_CHANGED, rand(), mesh->self->name);
+	send_request(mesh, mesh->everyone, mesh->self->submesh, "%d %x %s", KEY_CHANGED, rand(), mesh->self->name);
```

Comment:
Use `NULL` instead of `mesh->self->submesh`.

Reply:
I have changed it to NULL